tap_shiphero/client.py

```diff
@@ -23,8 +24,10 @@ def __exit__(self, type, value, traceback):
         self.__session.close()
 
     @backoff.on_exception(backoff.expo,
-                          [Server5xxError,
-                           RateLimitException],
+                          (Server5xxError,
+                           RateLimitException,
+                           exceptions.Timeout,
+                           exceptions.ConnectionError),
```

Comment:
Should this also include `exceptions.ChunkedEncodingError`? Looks like that was the final exception in the stack traces. And from the looks of it [here](https://github.com/requests/requests/blob/master/requests/exceptions.py#L92) it doesn't inherit from `ConnectionError`.

```
2018-12-28 18:27:00,886Z    tap -   File "/code/orchestrator/tap-env/lib/python3.5/site-packages/requests/models.py", line 753, in generate
2018-12-28 18:27:00,886Z    tap -     raise ChunkedEncodingError(e)
2018-12-28 18:27:00,886Z    tap - requests.exceptions.ChunkedEncodingError: ("Connection broken: ConnectionResetError(104, 'Connection reset by peer')", ConnectionResetError(104, 'Connection reset by peer'))
```